### ISSUE TITLE
Circular dependency between modV and modV-MediaManager

### ISSUE THREAD

**Issue by jackwilsdon** [vcync/modV#236]
This seems to cause `yarn` to get stuck when building, alternating between `yesno` and `vuex-localstorage`.

The issue also seems to exist on the v1.5 branch, where `yarn` results in this ever expanding output (going on forever);

```
warning modv-media-manager > ffbinaries@0.1.8: v1.0.0 More user-friendly syntax, improved reliability and error handling, added unit tests. Previous versions are not maintained anymore.
warning modv-media-manager > modv > modv-media-manager > ffbinaries@0.1.8: v1.0.0 More user-friendly syntax, improved reliability and error handling, added unit tests. Previous versions are not maintained anymore.
warning modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > ffbinaries@0.1.8: v1.0.0 More user-friendly syntax, improved reliability and error handling, added unit tests. Previous versions are not maintained anymore.
warning modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > ffbinaries@0.1.8: v1.0.0 More user-friendly syntax, improved reliability and error handling, added unit tests. Previous versions are not maintained anymore.
warning modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > ffbinaries@0.1.8: v1.0.0 More user-friendly syntax, improved reliability and error handling, added unit tests. Previous versions are not maintained anymore.
warning modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > ffbinaries@0.1.8: v1.0.0 More user-friendly syntax, improved reliability and error handling, added unit tests. Previous versions are not maintained anymore.
warning modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > ffbinaries@0.1.8: v1.0.0 More user-friendly syntax, improved reliability and error handling, added unit tests. Previous versions are not maintained anymore.
warning modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > ffbinaries@0.1.8: v1.0.0 More user-friendly syntax, improved reliability and error handling, added unit tests. Previous versions are not maintained anymore.
warning modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > ffbinaries@0.1.8: v1.0.0 More user-friendly syntax, improved reliability and error handling, added unit tests. Previous versions are not maintained anymore.
warning modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > ffbinaries@0.1.8: v1.0.0 More user-friendly syntax, improved reliability and error handling, added unit tests. Previous versions are not maintained anymore.
warning modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > ffbinaries@0.1.8: v1.0.0 More user-friendly syntax, improved reliability and error handling, added unit tests. Previous versions are not maintained anymore.
warning modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > ffbinaries@0.1.8: v1.0.0 More user-friendly syntax, improved reliability and error handling, added unit tests. Previous versions are not maintained anymore.
warning modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > ffbinaries@0.1.8: v1.0.0 More user-friendly syntax, improved reliability and error handling, added unit tests. Previous versions are not maintained anymore.
warning modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > ffbinaries@0.1.8: v1.0.0 More user-friendly syntax, improved reliability and error handling, added unit tests. Previous versions are not maintained anymore.
warning modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > ffbinaries@0.1.8: v1.0.0 More user-friendly syntax, improved reliability and error handling, added unit tests. Previous versions are not maintained anymore.
warning modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > ffbinaries@0.1.8: v1.0.0 More user-friendly syntax, improved reliability and error handling, added unit tests. Previous versions are not maintained anymore.
warning modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > ffbinaries@0.1.8: v1.0.0 More user-friendly syntax, improved reliability and error handling, added unit tests. Previous versions are not maintained anymore.
warning modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > ffbinaries@0.1.8: v1.0.0 More user-friendly syntax, improved reliability and error handling, added unit tests. Previous versions are not maintained anymore.
warning modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > ffbinaries@0.1.8: v1.0.0 More user-friendly syntax, improved reliability and error handling, added unit tests. Previous versions are not maintained anymore.
warning modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > modv > modv-media-manager > ffbinaries@0.1.8: v1.0.0 More user-friendly syntax, improved reliability and error handling, added unit tests. Previous versions are not maintained anymore.
```

**Comment by 2xAA** [comment]
Ah that's what that is.
npm had this also, but got fixed.

Will fix asap.

**Comment by jackwilsdon** [comment]
Much appreciated! Been playing around with the 1.5 branch and it's pretty cool :grin:

**Comment by 2xAA** [comment]
Nice - thanks for trying it out 😄 

`2.0-drawloop` could be considered the "edge" branch at the moment.
That's where the fun _really_ lies 😉

**Comment by 2xAA** [comment]
https://github.com/2xAA/modV-MediaManager/issues/11 should have fixed this - let me know!

A task I should really do is move the media manager to npm.
Will make an issue haha.

**Comment by jackwilsdon** [comment]
Looks like that's fixed it! I've been playing with `2.0-drawloop` and I love the new UI and features, it's incredible! I've noticed that some of the effects just show as black, is that a known issue?
